src/MongoDB.Bson/ObjectModel/ObjectId.cs

```diff
@@ -119,6 +120,13 @@ public ObjectId(string value)
             FromByteArray(bytes, 0, out _a, out _b, out _c);
         }
 
+        private ObjectId(int a, int b, int c)
+        {
```

Comment:
I didn't add any validation here since it's a private constructor, all possible validation should be done in higher levels